application/microservices/entando-hub-catalog-ms/src/main/java/com/entando/hub/catalog/rest/BundleGroupVersionController.java

```diff
@@ -80,7 +80,7 @@ public BundleGroupVersionController(BundleGroupVersionService bundleGroupVersion
     @ApiResponse(responseCode = "200", description = "OK")
     public ResponseEntity<BundleGroupVersion> createBundleGroupVersion(@RequestBody BundleGroupVersionView bundleGroupVersionView) {
         logger.debug("REST request to create BundleGroupVersion: {}", bundleGroupVersionView);
-        Optional<com.entando.hub.catalog.persistence.entity.BundleGroup> bundleGroupOptional = bundleGroupService.getBundleGroup(bundleGroupVersionView.getBundleGroupId().toString());
+        Optional<com.entando.hub.catalog.persistence.entity.BundleGroup> bundleGroupOptional = bundleGroupService.getBundleGroup(Long.parseLong(bundleGroupVersionView.getBundleGroupId()));
```

Comment:
Please add an an import for the BundleGroup class to simplify this declaration.

Reply:
As agreed during the meeting we decided to address these changes in a dedicated PR in the next days.